integration-tests/src/test/java/com/datastax/dse/driver/api/core/graph/remote/ClassicGraphTraversalRemoteIT.java

```diff
@@ -70,14 +70,12 @@ protected boolean isGraphBinary() {
 
   @Override
   protected GraphTraversalSource graphTraversalSource() {
-    return AnonymousTraversalSource.traversal()
-        .withRemote(DseGraph.remoteConnectionBuilder(session()).build());
+    return traversal().withRemote(DseGraph.remoteConnectionBuilder(session()).build());
   }
 
   @Override
   protected SocialTraversalSource socialTraversalSource() {
-    return EmptyGraph.instance()
-        .traversal(SocialTraversalSource.class)
+    return traversal(SocialTraversalSource.class)
```

Comment:
AnonymousTraversalSouce is [preferred](https://tinkerpop.apache.org/docs/current/upgrade/#_anonymoustraversalsource) over EmptyGraph in 3.5.0.  Once the static import came in the additional refactoring above followed naturally.